omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js

```diff
@@ -148,13 +150,14 @@ export default class ProgramStatusController {
     }
 
     toggleExpandedWorkflow(workflowUuid) {
-        // the first time we hit this, we need to initialize the workflow
-        if (!workflowUuid in this.expanded.workflow) {
-            this.expanded.workflow[workflowUuid] = true;
-        }
-        else {
-            this.expanded.workflow[workflowUuid] = !this.expanded.workflow[workflowUuid];
-        }
+        this.expanded.workflow[workflowUuid] = !this.expanded.workflow[workflowUuid];
+    }
+
+    // Sets workflowws to expanded display of containing states based on boolean configuration value - this.config.expandedWorkflow
+    expandWorkflows() {
```

Comment:
This method is really "setInitialExpandedStateForAllWorkflows", correct?  That's bit wordy, happy if we can come up with something shorter, but "expandWorkflows" seems incorrect and confusing (if I'm understanding what this is doing correctly, let me know if I'm wrong!)

Reply:
@Ruhanga there's also a typo in the comment above the new method ("workflowws").

Btw could this not have worked:
```javascript
toggleExpandedWorkflow(workflowUuid) {
    // the first time we hit this, we need to initialize the workflow
    if (!workflowUuid in this.expanded.workflow) {
        this.expanded.workflow[workflowUuid] = this.expandedWorkflow;
    }
    else {
        this.expanded.workflow[workflowUuid] = !this.expanded.workflow[workflowUuid];
    }
}
```
?

Also I would probably rename `this.expandedWorkflow` into something like `this.initExpandedWorkflow`.

Reply:
@mks-d, the above snippet executes when the `Show more` is first clicked and later. So upon first page loading this doesn't get hit.

Thanks, I'll rename `this.expandWorkflows` to `this.initExpandedWorkflow`. I guess that would be consistent with `this.toggleExpandedWorkflow`. cc: @mogoodrich.

Reply:
Yes, I like "initExpandedWorkflows", thanks @Ruhanga